tests/migration_test.go

```diff
@@ -3470,8 +3483,8 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				libnode.AddLabelToNode(sourceNode.Name, cleanup.TestLabelForNamespace(util.NamespaceTestDefault), "target")
 
 				By("starting four VMIs on that node")
-				for _, vmi := range vmis {
-					_, err := virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(vmi)
+				for i := range vmis {
+					vmis[i], err = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(vmis[i])
```

Comment:
This change does not seem to be related to this commit.

Reply:
Yes it's needed to pass the test, otherwise we have no namespace set inside the vmi struct.